mico-core/src/main/java/io/github/ust/mico/core/service/MicoKubernetesClient.java

```diff
@@ -227,9 +238,8 @@ public boolean isApplicationDeployed(MicoApplication micoApplication) throws Kub
     }
 
     /**
-     * Looks up if the {@link MicoService} is already deployed to the Kubernetes cluster.
-     * If so, it returns the Kubernetes {@link Deployment} object.
-     * Labels are used for the lookup.
+     * Looks up if the {@link MicoService} is already deployed to the Kubernetes cluster. If so, it returns the
```

Comment:
Replace _Looks up_ by _Checks_. _If so, it returns [...]_ should be placed in the `@return` part of the comment. Return type is incorrect, it actually is an `Optional<Deployment`, also indicate that an empty `Optional` is returned in case deplopyment is found in Kubernetes.

Reply:
Local variable `result` can be omitted, use direct `return` statement instead.

Reply:
Check all other methods regarding the above mentioned points.